NPE in android.content.SyncRequest
ndroid.content.SyncRequest.Builder().setSyncAdapter(accountManagerAccounts[0],BuildConfig.PROVIDER_AUTHORITY).syncOnce().build());

 - What happened.
 The following NPE occur
 java.lang.IllegalArgumentException: error unparceling Bundle
       at android.content.ContentResolver.validateSyncExtrasBundle(ContentResolver.java:1835)
       at android.content.SyncRequest$Builder.build(SyncRequest.java:490)
       at com.couchsurfing.mobile.manager.SyncManager.requestSync(SyncManager.java:302)

diff --git a/core/java/android/content/SyncRequest.java b/core/java/android/content/SyncRequest.java
index 1008a5a..ff6f6bb 100644
--- a/core/java/android/content/SyncRequest.java
+++ b/core/java/android/content/SyncRequest.java
@@ -486,10 +486,11 @@ public Builder setExpedited(boolean expedited) {
          *         builder.
          */
         public SyncRequest build() {
-            // Validate the extras bundle
-            ContentResolver.validateSyncExtrasBundle(mCustomExtras);
             if (mCustomExtras == null) {
                 mCustomExtras = new Bundle();
+            } else {
+                // Validate the extras bundle
+                ContentResolver.validateSyncExtrasBundle(mCustomExtras);
             }
             // Combine builder extra flags into the config bundle.
             mSyncConfigExtras = new Bundle();
